### ISSUE TITLE
Number of alternative splices events detected by the pipeline is significantly less than the annotation file

### ISSUE THREAD

**Issue by weir12** [nanoporetech/pipeline-pinfish-analysis#9]
Hi:
  I have successfully run the entire pipeline.But I found the **number of alternative splices** events detected by the pipeline is significantly **less than the annotation** file by comparing `results/polished_transcripts_collapsed.gff` and `Araport11_GFF3_genes_transposons.201606.gtf`
  **Only 155 AS events** were detected in my sample, compared to 19,828 AS events in the annotation file.
Here is sample stat of my input fastq file(Nanopore direct RNA sequencing of arabidopsis).
```
file                    format  type   num_seqs        sum_len  min_len  avg_len  max_len
./my_sample.fastq     FASTQ   RNA   1,678,300  1,551,903,455      100    924.7   11,538
```

Thanks for helping !

**Comment by bsipos** [comment]
Hello,

By alternative splicing events I guess you mean isoforms. In any RNA sequencing application the number of observed isoforms will depend on the number of reads. That is, the number of observed isoforms will increase with the number of reads until it reaches saturation.
Since you have only about 1.5 million reads, you can expect to observe only a fraction of expressed isoforms. Also, if you set -c to 10 for example, pinfish will discard all isofroms with a coverage less than 10. Moreover, you will loose some isoforms during the collapsing steps.

Finally, the annotation is likely to contain isofroms from different tissues/life stages. You cannot expect to observe all of them in any single tissue sample.

In summary, you can observe more isoforms by sequencing more reads (preferred) or lowering -c.

Best,
Botond
